common/host/fetch_git.sh

```diff
@@ -18,7 +18,7 @@ if [[ -d "$git_path" ]]; then
     set +e
     if ! git checkout "$git_version"; then
         set -e
-        git fetch origin -a
+        git fetch origin --tags
```

Comment:
This is fine, though I think it restricts $git_version to only being tags or commits within a tag?